test/e2e/suite_test.go

```diff
@@ -138,6 +138,8 @@ var _ = BeforeSuite(func() {
 					"replicas=1",
 					"--set", fmt.Sprintf("hostname=%s.%s", e2eCfg.ExternalIP, e2eCfg.MagicDNS),
 					"--create-namespace",
+					"--devel",
+					"--set rancherImageTag=v2.9-head",
```

Comment:
as is it throws an error:
```
[flag: --set rancherImageTag
      : exit status 1:Error: unknown flag: --set rancherImageTag
      
      {](msg: "[helm -n cattle-system install --set bootstrapPassword=admin --set global.cattle.psp.enabled=false --set extraEnv[0].name=CATTLE_SKIP_HOSTED_CLUSTER_CHART_INSTALLATION --set-string extraEnv[0].value=true --set replicas=1 --set hostname=172.18.0.3.sslip.io --create-namespace --devel --set rancherImageTag=v2.9-head rancher https://releases.rancher.com/server-charts/latest/rancher-2.8.2.tgz] cmd, failed with the following error: Error: unknown flag: --set rancherImageTag\n",)c
```

to fix it, please change it to:

```suggestion
					"--set",
					"rancherImageTag=v2.9-head",
```

Reply:
I fixed it once again, now it is ok